CHANGES.rst

```diff
@@ -12,6 +12,7 @@ Version 0.8.0 (UNRELEASED)
     - Adds a default ``kubernetes_memory_limit`` value (4Gi).
     - Adds configuration environment variable to set workflow scheduling policy (``REANA_WORKFLOW_SCHEDULING_POLICY``).
     - Changes Helm templates to use PostgreSQL 12.8 version.
+    - Adds configuration environment variable to set API limiter configs (``REANA_RATELIMIT_GUEST_USER``, ``REANA_RATELIMIT_AUTHENTICATED_USER``).
```

Comment:
We usually group changelog items per verb, i.e. All the "Adds..." together, all the "Changes..." and so on.

Reply:
Done

Reply:
Not sure if the word "configs" here is the more appropriate one. We can be more descriptive as this is targeted at administrators.

Reply:
I agree. Not sure about what can be a better name. Some options:

- `...API rate limit values`
- `...API rate limiting characteristics` (taken from `flask-limiter` docs page)

P.S need to change `variable` to `variables` :)

Reply:
"Adds configuration environment variables to set REST API rate limit values (VAR1, VAR2)" looks good...

Reply:
Done